webflow-hacks/misc-hacks/20/20.ts

```diff
@@ -0,0 +1,39 @@
+document.addEventListener('DOMContentLoaded', function () {
+  const UL_SELECTOR = '[fs-hacks-element="unordered-list"]';
+  const originalList = document.querySelector(UL_SELECTOR);
+  const unorderedList = document.createElement('ul');
```

Comment:
What about ordered lists? No support for them?

Reply:
Perhaps a new hack for this?

Reply:
Hm... feels like we would be repeating too much code.
What would be the implementation of nested ordered lists?
How would the nested bullets be represented? Numbers? Letters? We need more insights.

Reply:
How about 1,2,3, then 3.1, 3.2 for sub bullets, then 3.1.1 for sub-sub bullet

Reply:
Works for me!